### PR TITLE
minor telemetry changes.

### PR DESCRIPTION
**Pull request by omeshp**

### CODE REVIEW

File: Extensions/ArtifactEngine/Engine/cilogger.ts [hunk 1]
```diff
@@ -6,15 +6,15 @@ var packagejson = require('../package.json');
...
-        taskDisplayName: tl.getVariable('Task.DisplayName'),
-        jobid: tl.getVariable('System.Jobid'),
-        releaseid: tl.getVariable('Release.ReleaseId'),
-        builduri: tl.getVariable('Build.BuildUri'),
-        buildid: tl.getVariable('Build.Buildid'),
-        agentVersion: tl.getVariable('Agent.Version'),
+        hostType: hostType,
+        definitionName: hostType === 'release' ? tl.getVariable('RELEASE.DEFINITIONNAME') : tl.getVariable('BUILD.DEFINITIONNAME'),
+        processId: hostType === 'release' ? tl.getVariable('RELEASE.RELEASEID') : tl.getVariable('BUILD.BUILDID'),
+        processUrl: hostType === 'release' ? tl.getVariable('RELEASE.RELEASEWEBURL') : (tl.getVariable('SYSTEM.TEAMFOUNDATIONSERVERURI') + tl.getVariable('SYSTEM.TEAMPROJECT') + '/_build?buildId=' + tl.getVariable('BUILD.BUILDID')),
```
Comment (GitHubSriramB) [inline Extensions/ArtifactEngine/Engine/cilogger.ts]: there is no Build.BuildUri variable?

File: Extensions/ArtifactEngine/Engine/cilogger.ts [hunk 2]
```diff
@@ -6,15 +6,15 @@ var packagejson = require('../package.json');
...
     return {
-        serverurl: tl.getVariable('System.TEAMFOUNDATIONSERVERURI'),
-        releaseurl: tl.getVariable('Release.ReleaseWebUrl'),
-        taskDisplayName: tl.getVariable('Task.DisplayName'),
-        jobid: tl.getVariable('System.Jobid'),
-        releaseid: tl.getVariable('Release.ReleaseId'),
-        builduri: tl.getVariable('Build.BuildUri'),
-        buildid: tl.getVariable('Build.Buildid'),
-        agentVersion: tl.getVariable('Agent.Version'),
+        hostType: hostType,
```
Comment (GitHubSriramB) [inline Extensions/ArtifactEngine/Engine/cilogger.ts]: serverUrl is not required? it would work for both build & release

**Review by GitHubSriramB — APPROVED** [review]

File: Extensions/ArtifactEngine/Engine/cilogger.ts (hunk 2 above)
Comment (omeshp) [reply Extensions/ArtifactEngine/Engine/cilogger.ts]: I need project information also in the url.

File: Extensions/ArtifactEngine/Engine/cilogger.ts (hunk 1 above)
Comment (omeshp) [reply Extensions/ArtifactEngine/Engine/cilogger.ts]: no, build url is not there.

**Pull request merged**